pkg/registry/registry_to_model.go

```diff
@@ -0,0 +1,241 @@
+package registry
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/operator-framework/operator-registry/pkg/model"
+	"github.com/operator-framework/operator-registry/pkg/property"
+)
+
+func CreateModelFromBundle(b *Bundle) (*model.Model, error) {
+	desc, err := b.csv.GetDescription()
+	if err != nil {
+		return nil, fmt.Errorf("Could not get description from bundle CSV:%s", err)
+	}
+	i, err := b.csv.GetIcons()
+	if err != nil {
+		return nil, fmt.Errorf("Could not get icon from bundle CSV:%s", err)
+	}
+	mIcon := &model.Icon{
+		// looks like the CSV icon is a slice of objects https://github.com/operator-framework/api/blob/master/pkg/operators/v1alpha1/clusterserviceversion_types.go#L283
+		// Any better idea here than just taking the first element of the slice and working with that? Since I'm assuming in most cases users only pass in one element (icon)
+		// in the CSV anyway
+		MediaType: i[0].MediaType,
+		Data:      []byte(i[0].Base64data),
+	}
+	pkg := &model.Package{
+		Name:        b.Annotations.PackageName,
+		Description: desc,
```

Comment:
Is the `declcfg.Package.Description` field meant to hold the huge long csv description? (I need to double check to see what I did when converting from the sqlite DB into the model).

Reply:
It is